op-node/rollup/derive/deposit_source_test.go

```diff
@@ -34,3 +35,17 @@ func TestEcotone4788ContractSourceHash(t *testing.T) {
 
 	assert.Equal(t, expected, actual.Hex())
 }
+
+// TestAfterForceIncludeSourceHash
+// cast keccak $(cast concat-hex 0x0000000000000000000000000000000000000000000000000000000000000003 $(cast keccak 0x01))
+// # 0x8afb1c4a581d0e71ab65334e3365ba5511fb15c13fa212776f9d4dafc6287845
+func TestAfterForceIncludeSource(t *testing.T) {
+	source := AfterForceIncludeSource{
```

Comment:
nit: this name is not consistent, its after force include, which iirc we moved away from

Reply:
I think we agreed to go with `AfterForceInclude`, but I'm happy to re-name it to a better name :)